client/components/volunteers/RockUpVolunteerForm/AddVolunteerForm.test.js

```diff
@@ -1,43 +1,71 @@
 import React from 'react'
-import { render, screen, fireEvent } from '@testing-library/react'
+import { render, screen, waitFor } from '@testing-library/react'
 import { Provider } from 'react-redux'
 import store from '../../../store'
 import AddVolunteerForm from './AddVolunteerForm'
 
 import { addVolunteer } from './AddVolunteerFormHelper'
+import userEvent from '@testing-library/user-event'
 
 jest.mock('./AddVolunteerFormHelper')
 
+describe('Add volunteer form can validate inputs', () => {
+  it('Should display "required" for empty required entires', async () => {
+    const handleSubmit = jest.fn()
+    render(<AddVolunteerForm onSubmit={handleSubmit}/>)
+
+    userEvent.clear(screen.getByLabelText(/firstName/i))
+    userEvent.clear(screen.getByLabelText(/lastName/i))
+
+    userEvent.click(screen.getByRole('button', { name: /add/i }))
+
+    const element = await screen.findAllByText('Required')
+
+    expect(addVolunteer).not.toHaveBeenCalled()
```

Comment:
I don't think this is asserting anything important. The only reason it isn't being called is because you're mocking out the event handler. I would remove this line and line 7. I don't think there is any reason you need to import `addVolunteer`.

Reply:
Updated